stable/satisfy/templates/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "satisfy.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "satisfy.name" . }}
+    helm.sh/chart: {{ include "satisfy.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "satisfy.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "satisfy.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
+      {{- with .Values.image.pullSecrets }}
+      imagePullSecrets:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      initContainers:
+        - name: init
+          image: "{{ include "satisfy.image" . }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/sh
+            - -c
+            - >
+              set -x;
+              set -e;
+              if [[ -e /app_persist/satis.json ]]; then
+                echo >&2 "Existing installation found, moving on..";
+                exit 0
+              else
+                echo >&2 "Seeding new instance..";
+                cp -r /app/* /app_persist/;
+                chown -R satisfy:satisfy /app_persist;
+              fi
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: app-persist
+              mountPath: /app_persist
+      containers:
+        - name: satisfy
+          image: "{{ include "satisfy.image" . }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          env:
+            - name: REPO_NAME
+              value: "{{ .Values.satisfy.repo_name }}"
+            - name: HOMEPAGE
+              value: "{{ .Values.satisfy.homepage }}"
+            {{- if .Values.satisfy.ssh_private_key }}
+            - name: SSH_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "satisfy.fullname" . }}
+                  key: SSH_PRIVATE_KEY
+            {{- end }}
+          {{- if .Values.livenessProbe.enabled }}
```

Comment:
Why do you make probes optional?

Reply:
I like being able to disable probes, as it often help with debugging. I really wish all charts gave that ability out of the box.